WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainCoordinator.swift

```diff
@@ -141,20 +134,39 @@ class RegisterDomainCoordinator {
 
     // MARK: Helpers
 
+    private func purchaseDomain(for site: Blog?, on viewController: UIViewController, completion: @escaping (Result<Void, Swift.Error>) -> Void) {
```

Comment:
I think a VM would be a better fit for these service calls but it doesn't have to be part of this PR.